skin.json

```diff
@@ -41,6 +41,21 @@
       }
     ]
   },
+  "languageDirections": {
```

Comment:
Can we add these properties to the `skin-schema.json`?

Reply:
\+